worldengine/cli/main.py

```diff
@@ -302,6 +306,8 @@ def main():
     g_generate.add_argument('--not-fade-borders', dest='fade_borders', action="store_false",
                                help="Not fade borders",
                                default=True)
+    g_generate.add_argument('--scatter', dest='scatter_plot',
```

Comment:
We could leave it as an option of the generation process for now but just change the name